lms/djangoapps/courseware/models.py

```diff
@@ -248,3 +255,113 @@ class Meta(object):   # pylint: disable=missing-docstring
 
     field = models.CharField(max_length=255)
     value = models.TextField(default='null')
+
+
+SCORE_CHANGED = Signal(
+    providing_args=[
+        'points_possible',  # Maximum score available for the exercise
+        'points_earned',   # Score obtained by the user
+        'user',  # User object representing the student
+        'course',  # CourseLocator object representing the course
+        'usage'  # BlockUsageLocator indicating the courseware instance
```

Comment:
I'll make that change (which means I won't need to convert the keys, although the wiki page you pointed to will be useful going forward). 

I'm going to be looking into the celery task next. Is there any documentation about how to configure celery and rabbitmq on the devstack?